tasks/run-matlab-tests/v0/task.json

```
@@ -12,28 +12,72 @@
         "Minor": 1,
         "Patch": 0
     },
+    "groups": [
+        {
+            "name": "testArtifact",
+            "displayName": "Generate Test Artifacts"
+        },
+        {
+            "name": "coverageArtifacts",
+            "displayName": "Generate Coverage Artifacts"
+        },
+        {
+          "name": "filterTests",
+          "displayName": "Filter Tests"
+        }
+    ],
     "inputs": [
         {
             "name": "codeCoverageCobertura",
             "type": "string",
             "label": "Cobertura code coverage",
             "defaultValue": "",
             "required": false,
+            "groupName": "coverageArtifacts",
             "helpMarkDown": "Path to write code coverage report in Cobertura XML format."
         },
+        {
+            "name": "modelCoverageCobertura",
+            "type": "string",
+            "label": "Cobertura model coverage",
+            "defaultValue": "",
+            "groupName": "coverageArtifacts",
+            "required": false,
+            "helpMarkDown": "Path to write model coverage report in Cobertura XML format."
+        },
+        {
+            "name": "testResultsSimulinkTest",
+            "type": "string",
+            "label": "Simulink test results",
+            "defaultValue": "",
+            "groupName": "testArtifact",
+            "required": false,
+            "helpMarkDown": "Path to export Simulink Test Manager results in MLDATX format."
+        },
+        {
+            "name": "testResultsPDF",
+            "type": "string",
+            "label": "Test results in PDF format",
+            "defaultValue": "",
+            "groupName": "testArtifact",
+            "required": false,
+            "helpMarkDown": "Path to write test results in PDF format."
+        },
         {
             "name": "selectByFolder",
             "type": "string",
-            "label": "Select by folder",
+            "label": "By folder name",
```

Think "name" may be extraneous here.
```suggestion
            "label": "By folder",
```

---

Done.